src/components/common/filters/DateRangeFilter.tsx

```diff
@@ -0,0 +1,286 @@
+import {
+  Dispatch,
+  FC,
+  SetStateAction,
+  useCallback,
+  useEffect,
+  useState,
+} from "react";
+import { Box, Grid } from "@mui/material";
+import dayjs, { Dayjs } from "dayjs";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import {
+  border,
+  borderRadius,
+  color,
+  fontColor,
+  padding,
+} from "../../../styles/constants";
+import { dateDefault } from "../constants";
+import { ParameterState } from "../store/componentParamReducer";
+import TimeRangeBarChart from "../charts/TimeRangeBarChart";
+import { OGCCollections } from "../store/OGCCollectionDefinitions";
+import { useDispatch } from "react-redux";
+import { AppDispatch } from "../store/store";
+import { fetchResultNoStore } from "../store/searchReducer";
+import { cqlDefaultFilters } from "../cqlFilters";
+import PlainDatePicker from "../datetime/PlainDatePicker";
+import PlainSlider from "../slider/PlainSlider";
+
+const datePikerSlotProps = {
```

Comment:
This file is confusing for me. Starting from here there are 35 duplicated codes; The component in this file is not consistent with the file name (`DateRangeSlider` vs `DateRangeFilter`); the name `DateRangeSlider` is also the same with another component. Please double-check this file

Reply:
Sorry for making you confused. Since this component is developed based on DateRangeSlider(which is used in detail page, only slider), I forgot to change the component name to DateRangeFilter(which is used in filter, including a slider and a time-range-chart).